browser/ui/views/brave_default_browser_dialog_view.cc

```diff
@@ -163,4 +168,9 @@ void BraveDefaultBrowserDialogView::OnAcceptButtonClicked() {
   // and it will be automatically freed once all its tasks have finished.
   base::MakeRefCounted<shell_integration::BraveDefaultBrowserWorker>()
       ->StartSetAsDefault(base::NullCallback());
```

Comment:
I think we've checked the result from this callback in FirstRunDialog. How about doing the same here?

Reply:
I'm thinking to change FirstRunDialog like this PR because we want to pin Brave as fast as possible.
If the user has shown a willingness to make Brave as a default browser,
I think pin to taskbar regardless of the result of default browser setting seems ok. WDYT?

Reply:
Changed as you suggested..I think setting as a default browser will not give failure.

Reply:
> If the user has shown a willingness to make Brave as a default browser,
> I think pin to taskbar regardless of the result of default browser setting seems ok. WDYT?


Yeah, I don't mind either way. I just thought it'd be better to stick to the same way.